tests/integration/questionnaire/test_questionnaire_endpoints.py

```diff
@@ -36,15 +36,17 @@ def test_post_invalid_questionnaire_location_raises_404(self):
         # Then I am shown a 404 page
         self.assertStatusNotFound()
 
-    def test_post_on_questionnaire_route_without_hub_raises_404(self):
+    def test_post_on_questionnaire_route_without_hub_redirects_to_first_incomplete_location(
+        self,
+    ):
         # Given
         self.launchSurvey("test_textfield")
 
         # When
         self.post(url=f"/questionnaire/")
 
         # Then
-        self.assertStatusNotFound()
+        self.assertInUrl("name-block")
```

Comment:
Was this changed intentionally to not 404 on a POST to the questionnaire root?

Reply:
Yeah that bit of code was removed in one of the changes above